LedStrip.h

```diff
@@ -10,13 +10,26 @@
 #include <FastLED.h>
 FASTLED_USING_NAMESPACE
 
-
+//First port in DOF configtool goes to D5, second to D6 and so on
 #define DATA_PIN1   14 // D5
 #define DATA_PIN2   12 // D6 
 #define DATA_PIN3   13 // D7
 #define DATA_PIN4   15 // D8
 #define DATA_PIN5   5 // D1 
-#define DATA_PIN6   2 // D4
+#define DATA_PIN6   4 // D2
```

Comment:
A lot of preconfigured Wemos from Arnoz' shop are setup with DATA_PIN2 set to 2, maybe we should keep that that way.

Reply:
did the change, updated comments in ledstrip.h